col.h

```diff
@@ -1,6 +1,9 @@
+#ifndef COL_H_
+#define COL_H_
```

Comment:
If you have a header file that you import directly, but is also imported by another dependency (in this case `game`->`col` and `game`->`bird`->`col`), you get double definition errors. This statement makes sure each header file is only parsed once. But now that I think about it, that's exactly what `#pragma once` does. I'll update it, since that's cleaner.